README.md

```diff
@@ -16,7 +16,35 @@ This project is built using the [ASP.NET MVC](http://www.asp.net/mvc) web framew
    cd ivr-phone-tree-csharp
    ```
 
-2. Build the solution
+1. Build the solution
+
+1. Expose your application to the wider internet using [ngrok](http://ngrok.com). This step
+  is important because the application won't work as expected if you run it through
+  localhost.
+
+  To start using `ngrok` in our project you'll have execute to the following line in the _command prompt_.
+
+  ```
+  ngrok http 55585 -host-header="localhost:55585"
+  ```
+
+  Keep in mind that our endpoint is:
+
+  ```
+  http://<your-ngrok-subdomain>.ngrok.io/Conference/ConnectClient
+  ```
+
+  Remember to update the Local.config file with the generated <your-ngrok-subdomain>.
+
+1. Configure Twilio to call your webhooks
+
+  You will also need to configure Twilio to call your application when calls are received on your `TWILIO_NUMBER`. The voice url should look something like this:
```

Comment:
I think the use of `TWILIO_NUMBER` is misleading. It makes me think in some configuration variable or something similar.

We can reduce the confusion by using something like "... calls are received in your [*Twilio Number*](https://www.twilio.com/user/account/messaging/phone-numbers)."

I would appreciate hearing your opinion on this.